### ISSUE TITLE
Qwen-1.5-8x7B : r/LocalLLaMA

### ISSUE THREAD

**Issue by irthomasthomas** [irthomasthomas/undecidability#562]
- [ ] [Qwen-1.5-8x7B : r/LocalLLaMA](https://www.reddit.com/r/LocalLLaMA/comments/1atw4ud/qwen158x7b/)

# TITLE: Qwen-1.5-8x7B : r/LocalLLaMA

**DESCRIPTION:** "Qwen-1.5-8x7B

New Model
Someone created a sparse MoE Qwen model by merging and finetuning Qwen1.5-7B

**Model:** [https://huggingface.co/Crystalcareai/Qwen1.5-8x7b](https://huggingface.co/Crystalcareai/Qwen1.5-8x7b)

**Dataset:** [https://huggingface.co/datasets/Crystalcareai/MoD](https://huggingface.co/datasets/Crystalcareai/MoD)

**Thread:**

I'm excited to release a project I've been working on the last couple of weeks.

Qwen1.5-8x7b: [http://huggingface.co/Crystalcareai/Qwen1.5-8x7b](http://huggingface.co/Crystalcareai/Qwen1.5-8x7b)

And the accompanying dataset created with the intention of encouraging MoE models to organically develop their own experts: [http://huggingface.co/datasets/Crystalcareai/MoD](http://huggingface.co/datasets/Crystalcareai/MoD)

The purpose and intention behind this project is better detailed in the model/dataset card, but basically:

I curated a diverse dataset from the highest quality conversations I could find. It's actually great. All sources are included in the dataset card.

I then trained Qwen1.5-7b on a 100k subset over 4 epochs.

Took that and made a MoE using @maximelabonne 's lazymergekit, utilizing a random gate and no base model.

Trained that on another 351,000 pairs. I had planned on doing 4 full epochs, but @runpod_io had cuda errors in my machine 3x, expending the rest of my budget for the project after only 0.45/4 epochs.

**Good news:**

Model is surprisingly awesome even at such a (comparatively) small training set size. Reasoning compares with Mixtral in my (very basic) tests.

Will benchmark it properly once runpod situation gets sorted, and plan to finish the rest of the training.

Thank you to @Teknium1 , @jon_durbin , @erhartford , Maxime Labonne, and @chargoddard for their contributions to open source AI and making these processes accessible and transparent. And of course thank you to @MistralAI for inspiring this work and @alibaba_cloud for releasing the weights of the Qwen1.5 family.

Teknium and Eric Hartford have been especially helpful, answering questions with humility and generosity.

We're just getting started."

**URL:** [https://www.reddit.com/r/LocalLLaMA/comments/1atw4ud/qwen158x7b/](https://www.reddit.com/r/LocalLLaMA/comments/1atw4ud/qwen158x7b/)

#### {'label-name': 'MoE-model-training', 'label-description': 'Describes the process of training a Mixture of Experts (MoE) model using diverse datasets and innovative techniques.', 'confidence': 52.55}
